apps/omg_watcher/lib/omg_watcher/exit_processor/in_flight_exit_info.ex

```diff
@@ -92,13 +90,23 @@ defmodule OMG.Watcher.ExitProcessor.InFlightExitInfo do
         }
 
   def new_kv(
```

Comment:
I realized that the name doesn't tell me anything... Also why not add Specs & docs as it's public